acvm/src/pwg.rs

```diff
@@ -7,21 +7,33 @@ use acir::{
 };
 use std::collections::BTreeMap;
 
+use self::arithmetic::ArithmeticSolver;
+
 // arithmetic
 pub mod arithmetic;
 // Directives
 pub mod directives;
 // black box functions
+pub mod block;
 pub mod hash;
 pub mod logic;
 pub mod range;
 pub mod signature;
 pub mod sorting;
 
+// Returns the concrete value for a particular witness
+// Returns None if the witness has no assignment
+pub fn witness_to_value(
+    initial_witness: &BTreeMap<Witness, FieldElement>,
+    witness: Witness,
+) -> Option<&FieldElement> {
+    initial_witness.get(&witness)
+}
+
 // Returns the concrete value for a particular witness
 // If the witness has no assignment, then
 // an error is returned
-pub fn witness_to_value(
+pub fn witness_to_value_unwrap(
```

Comment:
prefixing a function name with unwrap has a particular meaning in Rust; reading the function name gives me the impression that inside it will be calling .unwrap and panicking. 

Why can't you use witness_to_value as it was when it was returning a Result, instead of creating this new function and having a different function that returns Option?

Reply:
Sure, it's done.